controllers/drinks.js

```diff
@@ -0,0 +1,53 @@
+const { Recipe } = require('../models/recipes');
+const { ctrlWrapper, isUserAdult } = require("../helpers")
+
+const getAllDrinksMainPage = async (req, res) => {
+    const birthDate = req.user.birthDate
+    const userIsAdult = isUserAdult(birthDate)
+    const findOptions = userIsAdult ? {} : {alcoholic:"Non alcoholic"}
+    const data = await Recipe.find(findOptions);
```

Comment:
The weight of request may be too big and take a time. Also according to tech task we should return cocktails separated into categories.